util/constants.ts

```diff
@@ -16,37 +16,37 @@ export enum SessionStorageKeys {
     PprWebUrl = 'PPR_WEB_URL',
     SiteminderLogoutUrl = 'SITEMINDER_LOGOUT_URL'
   }
-  
-  export enum Account {
+
+export enum Account {
     ANONYMOUS = 'ANONYMOUS',
```

Comment:
FWIW, most files use an indent of 2 spaces.

ESLint may fix this for you, or maybe prettier?

Reply:
implemented eslint requested changes in the entire project